### PR TITLE
Add Ko.md

### PR DESCRIPTION
**Pull request by kosyoz**

### CODE REVIEW

**Comment by JasmineSee** [comment]
Dummy comment

File: members/Ko.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## No Work Experience
+
+* [3 months: 2017 May - July] Sofware Engineering Intern, at PayPal Singapore
+* ...
+
+## Accomplishments
+
+1. Migrated large legacy system to a new platfrom: During my internship at PayPal, I single-handedly migrated a 20 KLoC software from platfrom X to Y. The migrated version is now live and used internally by 200+ employees.
+2. ...
```
Comment (zameschua) [inline members/Ko.md:13]: You need to add more accomplishments if you want to be accomplished in life

File: members/Ko.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Ko Szu Yuan
+
+## Poly Diploma
```
Comment (zameschua) [inline members/Ko.md:3]: Be more specific about the diploma here

File: members/Ko.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## No Work Experience
+
+* [3 months: 2017 May - July] Sofware Engineering Intern, at PayPal Singapore
```
Comment (zameschua) [inline members/Ko.md:7]: You said that you have no work experience but talk about Software Engineering Intern here??

**Review by zameschua — CHANGES_REQUESTED** [review]
Generally good forking workflow, but some minor problems with the contents.

**Pull request merged**